functions/src/authorization/index.ts

```diff
@@ -9,8 +9,13 @@ export const grantAdminRole = functions.https.onRequest((request: any, response:
           setAdminPriveleges(email).catch(error => {
               response.status(400).send("Error giving admin privileges:"+ error);
           })
-          if (decodedToken.isAdmin) { response.send("User has admin privileges");}
-          else { response.send("User does not have admin privileges");}
+          if (decodedToken.isAdmin) { 
+            var htmlString = '<label for = "userFile">Upload file </label>'
```

Comment:
this is sort of the same bad practice as returning a hardcoded string message. Returning html you expect the frontend to render or a string you expect the frontend to know/parse tightly couples the backend and frontend together.